README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
+   + I have recently come around to some genre permeability with fantasy
+      + If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
+  + My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
```

Comment:
This was a nice combination of using bold language, italics, and staggered bullets. It was also a nice way to drop bits about yourself in an easy to ready, quick snapshot.